src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/convolution_backprop_data.cpp

```diff
@@ -0,0 +1,118 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/convolution_backprop_data.hpp"
+#include "shared_test_classes/single_layer/convolution_backprop_data.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/filter_cpu_info.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+
+using namespace CPUTestUtils;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+
+namespace {
+
+/* INSTANCES */
+/* ============= Deconvolution (Planar 2D) ============= */
+const auto convParams_ExplicitPadding_Planar_2D = ::testing::Combine(
+        ::testing::ValuesIn(kernels2d),
+        ::testing::ValuesIn(strides2d),
+        ::testing::ValuesIn(padBegins2d),
+        ::testing::ValuesIn(padEnds2d),
+        ::testing::ValuesIn(dilations2d),
+        ::testing::ValuesIn(numOutChannels_Planar),
+        ::testing::Values(ngraph::op::PadType::EXPLICIT),
+        ::testing::ValuesIn(emptyOutputPadding)
+);
+
+const std::vector<DeconvInputData> Planar_2D_inputs_smoke = {
+        DeconvInputData{
+                InputShape{{}, {{ 2, 12, 7, 7 }}},
```

Comment:
Is it possible to add dynamic shapes to the arm testing as well?
I guess in this case the last two dimensions should just be equal, i.e. {-1, -1}, {7, 7}, {5. 5} etc.

Reply:
Applicable for all the instances where we test dynamic shapes in case of x86 and do not test for arm.

Reply:
created task - CVS-127974